### ISSUE TITLE
Use relative paths with Next

### ISSUE THREAD

**Issue by ctwhome** [research-software-directory/RSD-as-a-service#139]
To help to move around code and improve development experience I would like to have relative paths with aliases as defined in the next.js documentation [here](https://nextjs.org/docs/advanced-features/module-path-aliases)

```javascript
// tsconfig.json or jsconfig.json
{
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      "@/components/*": ["components/*"]
    }
  }
}
```

```javascript
// pages/index.js
import Button from '@/components/button'

export default function HomePage() {
  return (
    <>
      <h1>Hello World</h1>
      <Button />
    </>
  )
}
```

**Comment by ctwhome** [comment]
We made this in #130

**Comment by ruester** [comment]
Currently, without changing the tsconfig, it also works with using `~`, like so:

```javascript
import {app} from '~/config/app'
import useSnackbar from '~/components/snackbar/useSnackbar'
import ContentLoader from '~/components/layout/ContentLoader'
import ConfirmDeleteModal from '~/components/layout/ConfirmDeleteModal'
import {Contributor, ContributorProps} from '~/types/Contributor'
import {
  addContributorToDb, deleteContributorsById,
  getAvatarUrl, getContributorsForSoftware,
  prepareContributorData, updateContributorInDb
} from '~/utils/editContributors'
import useOnUnsaveChange from '~/utils/useOnUnsavedChange'
import {getDisplayName} from '~/utils/getDisplayName'
import {sortOnStrProp} from '~/utils/sortFn'
import {getPropsFromObject} from '~/utils/getPropsFromObject'
```
it would really improve the maintainability and readability of the code, especially in case of refactoring and moving components around this helps a lot :+1:

**Comment by dmijatovic** [comment]
This is indeed implemented #130, so I am closing this issue

**Comment by ruester** [comment]
But all the other components still use a lot of `../../../../`, this should be refactored too
